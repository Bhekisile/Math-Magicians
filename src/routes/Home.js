import loremIpsum from 'react-lorem-ipsum';
import '../styles/Home.css';

function Home() {
  return (
    <div>
      <h2 style={{ margin: '60px 140px' }}>Welcome to our page!</h2>
      <div className="text-wrapper">
        {loremIpsum({ avgSentencesPerParagraph: 10, random: true, p: 2 }).map((text) => (
          <div className="text" key={text}>
            {text}
          </div>
        ))}
      </div>
    </div>
  );
}

export default Home;
