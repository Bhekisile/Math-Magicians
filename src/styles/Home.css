.text-wrapper {
  width: 80%;
  margin: 10px auto;
}

.text {
  margin-bottom: 20px;
}
