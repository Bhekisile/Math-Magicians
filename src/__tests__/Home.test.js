import { render } from '@testing-library/react';
import Home from '../routes/Home';

test('renders text to homepage', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
