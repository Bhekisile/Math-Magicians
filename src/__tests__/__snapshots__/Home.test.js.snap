// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders text to homepage 1`] = `
<div>
  <div>
    <h2
      style="margin: 60px 140px;"
    >
      Welcome to our page!
    </h2>
    <div
      class="text-wrapper"
    >
      <p>
        Lorem ipsum dolor sit amet consectetur adipiscing elit. Ex sapien vitae pellentesque sem placerat in id. Pretium tellus duis convallis tempus leo eu aenean. Urna tempor pulvinar vivamus fringilla lacus nec metus. Iaculis massa nisl malesuada lacinia integer nunc posuere. Semper vel class aptent taciti sociosqu ad litora. Conubia nostra inceptos himenaeos orci varius natoque penatibus. Dis parturient montes nascetur ridiculus mus donec rhoncus. Nulla molestie mattis scelerisque maximus eget fermentum odio. Purus est efficitur laoreet mauris pharetra vestibulum fusce.
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipiscing elit. Ex sapien vitae pellentesque sem placerat in id. Pretium tellus duis convallis tempus leo eu aenean. Urna tempor pulvinar vivamus fringilla lacus nec metus. Iaculis massa nisl malesuada lacinia integer nunc posuere. Semper vel class aptent taciti sociosqu ad litora. Conubia nostra inceptos himenaeos orci varius natoque penatibus. Dis parturient montes nascetur ridiculus mus donec rhoncus. Nulla molestie mattis scelerisque maximus eget fermentum odio. Purus est efficitur laoreet mauris pharetra vestibulum fusce.
      </p>
    </div>
  </div>
</div>
`;
